interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -335,17 +339,27 @@ namespace cling {
       "\n"
       "   " << metaString << "U <filename>\t\t- Unloads the given file\n"
       "\n"
-      "   " << metaString << "I [path]\t\t\t- Shows the include path. If a path is given -"
+      "   " << metaString << "I [path]\t\t\t- Shows all include paths. If a path is given -"
+                             "\n\t\t\t\t  adds the path to the include paths\n"
+      "   " << metaString << "include [path]\t\t\t- Shows all include paths. If a path is given -"
                              "\n\t\t\t\t  adds the path to the include paths\n"
       "\n"
-      "   " << metaString << "O <level>\t\t\t- Sets the optimization level (0-3)"
-                             "\n\t\t\t\t  (not yet implemented)\n"
+      "   " << metaString << "O <level>\t\t\t- Prints the optimization level (0-3)"
+                             "\n\t\t\t\t  If a level is given, it sets it.\n"
```

Comment:
I would reverse both sentences, as in
```suggestion
      "   " << metaString << "O <level>\t\t\t- Sets the optimization level (0-3)"
                             "\n\t\t\t\t  If no level is given, print the current setting.\n"
```